README.md

```diff
@@ -713,6 +713,9 @@ model config, you are letting Triton decide the placement of input Tensors:
 parameters: { key: "FORCE_CPU_ONLY_INPUT_TENSORS" value: {string_value:"no"}}
 ```
 
+Using this configuration parameter, you **cannot** force your input tensors to
```

Comment:
This is still not clear... it doesn't describe what happens with FORCE is false. See above comment... you might want to remove this.